upgrade/upgrade.go

```diff
@@ -73,6 +73,26 @@ var (
 		EtnaTime:                     InitiallyActiveTime,
 	}
 
+	// LatestUnscheduled is the configuration that deactivates the latest
+	// non-scheduled upgrade. This configuration is safe for networks that requires
+	// backwards compatibility with the latest upgrade.
+	LatestUnscheduled = Config{
+		ApricotPhase1Time:            InitiallyActiveTime,
+		ApricotPhase2Time:            InitiallyActiveTime,
+		ApricotPhase3Time:            InitiallyActiveTime,
+		ApricotPhase4Time:            InitiallyActiveTime,
+		ApricotPhase4MinPChainHeight: 0,
+		ApricotPhase5Time:            InitiallyActiveTime,
+		ApricotPhasePre6Time:         InitiallyActiveTime,
+		ApricotPhase6Time:            InitiallyActiveTime,
+		ApricotPhasePost6Time:        InitiallyActiveTime,
+		BanffTime:                    InitiallyActiveTime,
+		CortinaTime:                  InitiallyActiveTime,
+		CortinaXChainStopVertexID:    ids.Empty,
+		DurangoTime:                  InitiallyActiveTime,
+		EtnaTime:                     UnscheduledActivationTime,
+	}
```

Comment:
Since this is only used in tests - I feel like using `upgradetest` would be preferred rather than adding this here. Also we could then use `GetConfig(Latest - 1)` rather than redefining this whole struct again.

Reply:
hmm I think that code was not there before. and I thought this can be used in elsewhere and can provide a reliable config after every new upgrade definition.

Reply:
changed it now with `GetConfig` from `upgradetest`